<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>setkey</class>
 <widget class="QDialog" name="setkey">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>253</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="setkey_button">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>140</y>
     <width>164</width>
     <height>18</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>80</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>40</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>appid</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>key</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>setkey_button</sender>
   <signal>rejected()</signal>
   <receiver>setkey</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>235</x>
     <y>157</y>
    </hint>
    <hint type="destinationlabel">
     <x>246</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>setkey_button</sender>
   <signal>accepted()</signal>
   <receiver>setkey</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>147</x>
     <y>157</y>
    </hint>
    <hint type="destinationlabel">
     <x>136</x>
     <y>195</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
